schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/index-fields/link-utility.xsl

```diff
@@ -75,22 +76,23 @@
 
 
   <xsl:variable name="documentsConfig" as="node()*">
-    <doc protocol="WWW:LINK" function="legend">
+    <doc protocol="WWW:LINK" function="legend" type="legend">
       <element>portrayalCatalogueCitation</element>
     </doc>
-    <doc protocol="WWW:LINK" function="featureCatalogue">
+    <doc protocol="WWW:LINK" function="featureCatalogue" type="fcats">
       <element>featureCatalogueCitation</element>
     </doc>
-    <doc protocol="WWW:LINK" function="dataQualityReport">
+    <doc protocol="WWW:LINK" function="dataQualityReport" type="dq-report">
       <element>additionalDocumentation</element>
       <element>specification</element>
       <element>reportReference</element>
     </doc>
   </xsl:variable>
 
   <xsl:template name="collect-documents">
-    <xsl:variable name="root" select="."/>
+    <xsl:param name="forIndexing" select="false()" as="xs:boolean"/>
```

Comment:
Can you add a code comment to explain what is the usage of this parameter?

Reply:
Added description https://github.com/geonetwork/core-geonetwork/pull/6588/commits/80fbd5f7da3ea517a32b9d711a41612693f61e14